Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -3136,7 +3136,7 @@ public void CalculateActionFinalStatus(Act act)
                     if (ve.Value.Contains("{Actual}"))
                     {
                         //Replace to 
-                        if ((actReturnValue.Actual != null) && Ginger.Utils.StringManager.IsNumeric(actReturnValue.Actual))
+                        if (!string.IsNullOrWhiteSpace(actReturnValue.Actual) && Ginger.Utils.StringManager.IsNumeric(actReturnValue.Actual))
```

Comment:
Whitespace will have " " in output value, output values needed in this format?

Reply:
.

Reply:
I tested the use cases where only whitespaces will be there in Output values and tried to evaluate them using CS operation, it works fine.
Though we could use IsNullOrEmpty() also, as in current issue we don't have a scenario for IsNullOrWhiteSpace.
Please suggest if we need to change this to IsNullOrEmpty().

Reply:
We are adding quotes for string values, for numeric its not being added, so empty or whitespace should also have quotes, let me know the use case for this

Reply:
User wants to see if the SQL Query output returns either null or some value, so when there is value the evaluate using CS works fine, but when the null is returned from DB, it fails.
user uses below CS for evaluating the value.
{CS Exp=({Actual}.Length == 0) || ({Actual}.Length == 2)}